spec/lib/vmdb/settings_spec.rb

```diff
@@ -566,6 +566,47 @@
       settings = Vmdb::Settings.for_resource(MiqServer.new(:zone => server.zone))
       expect(settings.api.token_ttl).to eq "4.hour"
     end
+
+    context "on global region" do
+      let(:key)                 { "yuri" }
+      let(:value_region_remote) { "region_remote" }
+      let(:value_server_remote) { "server_remote" }
+
+      before do
+        Zone.seed
+        @region_global = MiqRegion.first
+        @region_remote = FactoryBot.create(:miq_region, :id => ApplicationRecord.id_in_region(1, @region_global.region + 1))
+        @zone_remote   = FactoryBot.create(:zone, :id => ApplicationRecord.id_in_region(2, @region_remote.region))
+        @server_remote = FactoryBot.create(:miq_server, :zone => @zone_remote, :id => ApplicationRecord.id_in_region(1, @region_remote.region))
+        SettingsChange.create!(:id            => ApplicationRecord.id_in_region(1, @region_global.region),
+                               :resource_id   => @region_global.id,
+                               :resource_type => "MiqRegion",
+                               :key           => "/#{key}",
+                               :value         => "value from global region")
+      end
+
+      after do
+        SettingsChange.delete_all
```

Comment:
This is not necessary, since all database changes are cleaned up afterwards anyway.

Reply:
interestingly, test was failing without that, but not every time

Reply:
we shouldn't band aid over sporadic failures, but instead solve them.  Let's take it out and deal with it.  It's possible I broke something with the settings cache since I changed that recently.

Reply:
I did tried to solve, but without success :(,
OK, removing explicit clean-up

Reply:
sooo strange, no errors anymore on my local (run several times)